<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>721</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>708</width>
      <height>719</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="tab_faceSides">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>691</width>
       <height>571</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="BaseJoints">
      <attribute name="title">
       <string>Base Joints</string>
      </attribute>
      <widget class="QLabel" name="label_sideFace">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>381</width>
         <height>481</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../prefs/icons/OFR_icons/OFR_FaceSideIcon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_head">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>370</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Head</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_neck">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>480</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Neck</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_jaw">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>270</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Jaw</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_lipParent">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>360</y>
         <width>71</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Lip Parent</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_eye">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>190</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Eye</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbSide">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Side Label</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Labels can be checked under Extra Attributes in the Attributes tab.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>161</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1) Create joints in the locations as indicated by the image on the right. As with most other face rigs, the eye joint should be placed in the center of the eye ball and joints that naturally fall in the center, like the head and jaw joint, should be placed in the center of the head.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Instructions</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>2) One at a time, select each joint and use the buttons associated with each joint to label them. It is important to use the Side Label for the Eye Joint. </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="BaseCurves">
      <attribute name="title">
       <string>Base Curves</string>
      </attribute>
      <widget class="QLabel" name="label_frontFace">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>381</width>
         <height>491</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../OFR_icons/OFR_FaceFrontIcon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_brow">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>40</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Brow</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_squint">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>200</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Squint</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_nostril">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>230</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Nostril</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_laugh">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>260</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Laugh</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_cheek">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>350</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Cheek</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_upperLip">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>400</y>
         <width>71</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Upper Lip</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_lowerLip">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>450</y>
         <width>71</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Lower Lip</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Side Label</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Instructions</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1) Make your character mesh LIVE by selecting it and clikcing the last magnet symbol on the modeling tab. </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>2) Draw curves on one side fo the face as indicated by the image to the right. Drawn curves should stick to the Mesh's surface once it is LIVE.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>181</width>
         <height>151</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>3) One at a time, select each curve and use the buttons on the right to indicate which curve is which. MAKE SURE TO SPECIFY WHICH SIDE OF THE FACE THE CURVE IS ON BY USING THE DROP BOX ABOVE THE IMAGE. (This should be relative to the character's POV)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Labels can be checked under Extra Attributes in the Attributes tab.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbSide_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_upperEyelid">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>110</y>
         <width>71</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Upper Lid</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_lowerEyelid">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>150</y>
         <width>71</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Lower Lid</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_build">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>650</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Build Face</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmbSide</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>cmbSide_2</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbSide_2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>cmbSide</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
